.bumpversion.cfg

```diff
@@ -1,16 +1,16 @@
 [bumpversion]
-current_version = 0.1.0a1.dev
+current_version = 0.1.0a2.dev
 commit = False
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(0a)?(?P<patch>\d+)(\.(?P<release>[a-z]+))?
-serialize =
+serialize = 
```

Comment:
```suggestion
serialize =
```

Reply:
it was done by bump2version, I just ran: 
```
python .github/release.py patch --lower 3.7 --upper 3.10
```
and rebased to fix the docs

Reply:
Interesting. That tool was not even supposed to touch that line, right?

Reply:
I believe so